Sources/Socket.swift

```diff
@@ -136,35 +138,37 @@ public final class Socket {
         }
 
         if result == -1 {
-            throw Error.lastError
+            throw ZeroMqError.lastError
         }
 
         return message
     }
 
     public func receive(_ bufferSize: Int = 1024, mode: ReceiveMode = []) throws -> Data? {
```

Comment:
This API should be `receive(into: UnsafeMutableBufferPointer<UInt8>` such as [here](https://github.com/Zewo/Zewo/blob/master/Modules/Axis/Sources/Axis/Stream/Stream.swift#L11). On a related note, `Socket` should adopt `Axis.Stream` to gain a bunch of convenience methods and be able to interact with the rest of the Zewo ecosystem.